packages/core/src/credentialsV1/KiltCredentialV1.ts

```diff
@@ -8,43 +8,66 @@
 import { hexToU8a } from '@polkadot/util'
 import { base58Encode } from '@polkadot/util-crypto'
 
-import { JsonSchema } from '@kiltprotocol/utils'
-import { CType } from '@kiltprotocol/core'
+import { JsonSchema, SDKErrors } from '@kiltprotocol/utils'
 import type {
   ICType,
   ICredential,
   DidUri,
   IDelegationNode,
 } from '@kiltprotocol/types'
 
-import { CredentialMalformedError } from './errors.js'
+import * as CType from '../ctype/index.js'
 import { fromGenesisAndRootHash } from './KiltRevocationStatusV1.js'
-import {
-  DEFAULT_CREDENTIAL_CONTEXTS,
-  DEFAULT_CREDENTIAL_TYPES,
-  JSON_SCHEMA_TYPE,
-  KILT_ATTESTER_DELEGATION_V1_TYPE,
-  KILT_ATTESTER_LEGITIMATION_V1_TYPE,
-  KILT_CREDENTIAL_TYPE,
-  W3C_CREDENTIAL_TYPE,
-  spiritnetGenesisHash,
-} from './constants.js'
+import { W3C_CREDENTIAL_CONTEXT_URL, W3C_CREDENTIAL_TYPE } from './constants.js'
 import type {
   KiltAttesterDelegationV1,
   KiltAttesterLegitimationV1,
   KiltCredentialV1,
 } from './types.js'
 import {
+  KILT_ATTESTER_DELEGATION_V1_TYPE,
+  KILT_ATTESTER_LEGITIMATION_V1_TYPE,
   credentialIdFromRootHash,
   jsonLdExpandCredentialSubject,
+  spiritnetGenesisHash,
 } from './common.js'
+import { CTypeLoader, newCachingCTypeLoader } from '../ctype/CTypeLoader.js'
 
 export {
   credentialIdFromRootHash as idFromRootHash,
   credentialIdToRootHash as idToRootHash,
   getDelegationNodeIdForCredential as getDelegationId,
 } from './common.js'
 
+/**
+ * Credential context URL required for Kilt credentials.
+ */
+export const CONTEXT_URL = 'https://www.kilt.io/contexts/credentials'
```

Comment:
moved constants to the files to which they relate most to make the relationships of the them clearer